### PR TITLE
Best Title

### PR DESCRIPTION
**Pull request by mmmmmrob**

### CODE REVIEW

**Comment by mmmmmrob** [comment]
@jaimeiniesta I think this is good to go.

File: spec/meta_inspector/texts_spec.rb [hunk 1]
```diff
@@ -3,10 +3,41 @@
...
+      page.best_title.should == 'This title came from the first h1'
+    end
+
+    it "should choose the longest candidate from the available options" do
+      page = MetaInspector.new('http://example.com/title_best_choice')
+      page.best_title.should == 'This title came from the first h1 and should be the longest of them all, so should be chosen'
+    end
+
+    it "should strip leading and trailing whitespace and all line breaks" do
+
```
Comment (jaimeiniesta) [inline spec/meta_inspector/texts_spec.rb]: I think you forgot to implement this spec.  :)

**Comment by mmmmmrob** [comment]
Thanks for spotting that Jaime. Now fixed.

**Comment by jaimeiniesta** [comment]
:+1: merged and released. Thanks!

**Pull request merged**